app/services/CaptchaService.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import WebView from 'react-native-webview';
+
+const patchPostMessageJsCode = `(${String(function() {
+  // eslint-disable-next-line no-var
+  let originalPostMessage = window.ReactNativeWebView.postMessage;
+  let patchedPostMessage = function(message, targetOrigin, transfer) {
+    originalPostMessage(message, targetOrigin, transfer);
+  };
+  patchedPostMessage.toString = function() {
```

Comment:
What exactly is this code doing?

Reply:
@aputinski  This is standard way of getting a message from React Native web view. Pretty much all react native captcha implementation use the exact same approach. From the React Native web view, it will call `window.ReactNativeWebView.postMessage` to post data, it's kind of how that specific library is working